expr/functions/consolidateBy/function.go

```diff
@@ -26,6 +26,16 @@ func New(configFile string) []interfaces.FunctionMetadata {
 	return res
 }
 
+var ValidAggregateFunctions = map[string]struct{}{
+	"average": {},
+	"avg":     {},
+	"max":     {},
+	"min":     {},
+	"sum":     {},
+	"first":   {},
+	"last":    {},
+}
```

Comment:
"mean" should not be?

Reply:
According to the [documentation](https://graphite.readthedocs.io/en/stable/functions.html#graphite.render.functions.consolidateBy) these are all of them.